release_process.md

```diff
@@ -0,0 +1,324 @@
+- the factory code is deployed on mainnet on `sputnik-dao.near` account
+- the factory code was deployed only once on mainnet on [June 01, 2021 at 6:17:46pm](https://explorer.mainnet.near.org/transactions/HZZpqMjbhCPWF8n5DXLMq5WtpmASfXuDLtYF6vNsiD9U)
+- the factory code is deployed on testnet on `sputnikv2.testnet` account
+
+**DAOs**:
+- the DAOs code can be found inside `sputnikdao2/src`
+- the DAO code is deployed to several accounts that have the following format: `<dao_name>.sputnik-dao.near` for mainnet and `<dao_name>.sputnikv2.testnet` for testnet
+- see all the DAOs that live on mainnet by calling `near view sputnik-dao.near get_dao_list`
+- see all the DAOs that live on testnet by calling `near view sputnik-v1.testnet get_dao_list`
+
+### UI DApps built on top of the Sputnik smart contracts (from oldest to newest)
+- https://old.sputnik.fund/ -> it uses V1 smart contracts
+- https://www.sputnik.fund/ -> it uses V1 smart contracts
+- https://v2.sputnik.fund/ -> it uses v2 smart contracts
+- https://astrodao.com/ -> it uses v2 smart contracts
+
+***
+
+## Introducing Sputnik v3 smart contracts
+
+The biggest advantage of v3 smart contracts is introducing an easy way for the DAO to upgrade to a new version of the code so it can take full advantage of the new features/performance improvements/bug fixes.  
+
+Since this is the first time that the factory and the DAO are being upgraded and upgrading smart contracts is a very sensitive topic, everything must be done with due diligence.
+
+### v3 Release Plan
+
+[1 Upgrade the factory from v2 to v3 and then set up the default code for the DAO to be v2.](#1-upgrade-the-factory-from-v2-to-v3)  
+[2 After we get enough confidence using factory v3 and DAO v2, change the default code for the DAO from v2 to v3.](#2-change-dao-default-code-from-v2-to-v3)  
+[3 Existing DAOs will need to migrate from v2 to v3.](#3-migrate-dao-from-v2-to-v3)  
+
+### 1 Upgrade the factory from v2 to v3
+
+This should be done in the following order:
+- [testnet - using a personal account](#1-1-using-personal-account-on-testnet)
+- [testnet - using the official testnet factory account](#1-2-using-official-account-on-testnet)
+- [mainnet - using the official mainnet factory account](#1-3-using-official-account-on-mainnet)
+
+___
+
+### 1 1 Using personal account on testnet
+
+***1. Create a new NEAR account for the factory:***
+
+```bash
+near create-account sputnik-factory.ctindogaru.testnet --masterAccount ctindogaru.testnet --initialBalance 50
+```
+
+***2. Deploy the factory code:***
+```bash
+./build.sh
+```
+```bash
+near deploy sputnik-factory.ctindogaru.testnet sputnikdao-factory2/res/sputnikdao_factory2.wasm
+```
+
+***3. Init the factory:***
+```bash
+near call sputnik-factory.ctindogaru.testnet new '{}' --accountId sputnik-factory.ctindogaru.testnet --gas 100000000000000
+```
+
+***4. Download the current `wasm` code used for creating new DAOs:***
+
+```bash
+near view sputnikv2.testnet get_dao_list
```

Comment:
Need to remove this command since the list is so big that it fails. Will redirect people to the astro testnet instead.

Reply:
Was gunna bring that up. I think if we do want to go this route, we have to do pagination. The other function should be the path devs use tho, since it already has pagination